<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDialogArticle</class>
 <widget class="QDialog" name="CInputDialogArticle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/icon.ico</normaloff>res/icon.ico</iconset>
  </property>
  <widget class="QLineEdit" name="lineEditMaxInventory">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikelbezeichnung:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kommentar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>1.Artikelnummer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Verkaufspreis:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWaregroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonWaregroup">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>390</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Lagerkapazität:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Währung:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>488</y>
     <width>981</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Einheit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Hersteller:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>2.Artikelnummer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Bestand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUnit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInventory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Einkaufspreis:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Standort:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxWarning">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Warengruppe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditArticlenumber_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>481</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValuta">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWarningLimit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditArticlenumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>481</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>981</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBaseprice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Warnung(wenn Bestand kleiner als):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSalesprice">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>abbrechen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>hinzufügen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditArticlenumber</tabstop>
  <tabstop>lineEditArticlenumber_2</tabstop>
  <tabstop>lineEditInventory</tabstop>
  <tabstop>lineEditMaxInventory</tabstop>
  <tabstop>checkBoxWarning</tabstop>
  <tabstop>lineEditWarningLimit</tabstop>
  <tabstop>lineEditUnit</tabstop>
  <tabstop>lineEditLocation</tabstop>
  <tabstop>lineEditBaseprice</tabstop>
  <tabstop>lineEditSalesprice</tabstop>
  <tabstop>lineEditValuta</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEditWaregroup</tabstop>
  <tabstop>pushButtonWaregroup</tabstop>
  <tabstop>lineEditComment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
